wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandScheduleTest.java

```diff
@@ -109,6 +111,102 @@ void schedulerCancelTest() {
     }
   }
 
+  @Test
+  void cancelNextCommandTest() {
+    try (CommandScheduler scheduler = new CommandScheduler()) {
+      Command[] commands = new Command[2];
+      var commandRunCounter = new AtomicInteger(0);
+      // due to the ordering of sets being non-deterministic, we cancel the other command in the
+      // first command and check that the second command is not run
```

Comment:
This is just me being super nit-picky:
```suggestion
      // for parity with C++ where the ordering of sets is non-deterministic, we cancel the other command in the
      // first command and check that the second command is not run
```
(Formatting probably needs to be changed)

In Java, the ordering of the particular kind of set we use to track commands ([`LinkedHashSet`](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/LinkedHashSet.html)) is actually deterministic (it's defined to be insertion order). However, it is true that the hashing method used by `SmallPtrSet` in heap allocation mode is non-deterministic (it's based on the pointer's numerical value), so the test should remain written the way it is.

Reply:
I fully agree with the nitpick, I should have been more specific about it, thx for noticing it

# added in 93ad7b8